clients/slack/schema.py

```diff
@@ -0,0 +1,8 @@
+from typing import Optional
+from pydantic import BaseModel
+
+
+class SlackMessageSchema(BaseModel):
+    text: Optional[str] = None
+    attachments: Optional[list] = None
```

Comment:
In the future we will create schemas for the attachments and the blocks as well

Reply:
The only concern I have with it, is that it's a new requirement and if it's not in use right now I'd rather avoid adding requirements. Also, if you decide to keep it - add it to the requirements and setup files

Reply:
Well I do use it :) 
Just not implemented the attachments schema all the way down, but we now have a schema for slack message which I think is good and useful :)